nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPath.java

```diff
@@ -255,6 +256,38 @@ public static AccessPath getAccessPathForNodeWithMapGet(Node node, @Nullable Typ
     }
   }
 
+  /**
+   * Gets corresponding AccessPath for a class field element.
+   *
+   * @param field The class field element.
+   * @param receivers The list of receivers.
+   * @return corresponding AccessPath.
+   */
+  public static AccessPath fromFieldAndBase(Element field, List<Element> receivers) {
+    List<AccessPathElement> elements = new ArrayList<>();
+    elements.add(new AccessPathElement(field));
+    Root root;
+    Element rootReceiver;
+    if (receivers == null || receivers.size() == 0) {
+      root = new Root();
+    } else {
+      rootReceiver = receivers.get(0);
+      assert !(rootReceiver instanceof MemberSelectTree);
```

Comment:
I don't understand this assertion.  Is a `MemberSelectTree` even an `Element`?  In any case, we should document in the Javadoc the exact condition that must hold for the 0th receiver, and then check that exact condition here, with a proper message if the check fails

Reply:
I made a mistake here in doing what I really wanted to do. My main intention was to prevent adding incomplete access paths to the store. Suppose that we have ```a.b.c```, this check was supposed to prevent adding only the partial access paths such as ```b.c``` to the store and only accept the complete access path ```a.b.c```. After giving it some thoughts, I think we should delegate the responsibility of access path completeness check, to the users of this method rather than this method itself.